apps/arm/main.cpp

```diff
@@ -206,11 +269,9 @@ static CANMsg::CANMsgHandlerMap canHandlerMap = {
 
     {CANID::SET_PID_TUNING_MODE,        &setPIDTuningMode},
```

Comment:
I mentioned this originally but forgot to remind you, the idea behind this command is to toggle in and out of PID tuning mode, so that changes to the constants can't be made while doing other operations with the arm. I think that this is important to consider but can be implemented in a separate PR if desired.

Reply:
what do you mean by "other operations to the arm"

Reply:
ie. the whole arm (PID and open loop control) should really be stopped while changing PID parameters, so that the operator can update them in a controlled way and have time to react after changing the tuning.

Reply:
I will create an issue for this. I dont think i'll get to it in this PR.